tests/httpwrappers/tests.py

```diff
@@ -746,6 +746,13 @@ def test_invalid_cookies(self):
         # document.cookie parses whitespace.
         self.assertEqual(parse_cookie('  =  b  ;  ;  =  ;   c  =  ;  '), {'': 'b', 'c': ''})
 
+    def test_samesite(self):
+        c = SimpleCookie('name=value; samesite=lax; httponly')
+        # this tests both the patched version of SimpleCookie and Morsel
+        # expected output:
+        # 'Set-Cookie: name=value; HttpOnly; SameSite=lax'
+        self.assertIn('SameSite', c.output())
```

Comment:
I think the comment doesn't add much value (and it'll be something we'd want to remember to remove eventually).

For the assertions, I'd use:
```
self.assertEqual(c['name']['samesite'], 'lax')
self.assertIn('SameSite=lax', c.output())
```